tests/e2e/e2e_common.go

```diff
@@ -32,6 +32,7 @@ const (
 	apiServerIPs                               = "API_SERVER_IPS"
 	attacherContainerName                      = "csi-attacher"
 	busyBoxImageOnGcr                          = "harbor-repo.vmware.com/csi/busybox:1.35"
+	windowsLTSC2019Image                       = "harbor-repo.vmware.com/csi/windows-servercore:ltsc2019"
```

Comment:
Point to Remember : going forward we should not using windowsLTCS due to heavy size of the image